mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/config/MasterConfiguration.java

```diff
@@ -351,6 +351,11 @@ default Duration getSchedulerIntervalBetweenRetries() {
     @Default("false")
     boolean isBatchSchedulingEnabled();
 
+    // Example: "jdk:17"
+    @Config("mantis.scheduler.schedulingConstraints")
+    @Default("")
+    String getSchedulingConstraints();
+
```

Comment:
Is it possible to perform the transformation at this level instead of the leaf level (i.e., the actual usage)? This will allow us to set the expectation that this should be the return type of the config value in case we switch from the config-magic library to a different one in the future. For example, take a look at the getHeartbeatInterval example below.